tasks/contrib-requirejs.js

```diff
@@ -3,11 +3,11 @@ module.exports = function(grunt) {
   grunt.config('requirejs', {
     public: {
       options: {
-        baseUrl: '.',
+        baseUrl: 'src',
```

Comment:
Why do you want to remove `src` from the paths in the compiled js file? Seems like a change that is prone to potential error through omission of paths that are missing this change in this PR. I think this change is fine otherwise, I'm just not immediately recognizing the reward part of risk / reward.

Reply:
This was necessary to maintain support for both "development" and "production" environments using paths as they are written in the source files. I see the value in simplifying the situation, but that entails touching the source code even more. That's done, now.